### PR TITLE
Pass sponsor and projectnumber through

### PR DESCRIPTION
**Pull request by kylebaron**

### CODE REVIEW

**Comment by andersone1** [comment]
@kylebaron  - is this ready for review?

**Comment by kylebaron** [comment]
@andersone1 It's ready; I was going to write a summary into the PR but didn't get to it yet.  Usually, I've been writing "WIP" in the title to mean not ready yet and then taking that off and requesting review when it is ready. This was sort of none of the above.

**Comment by kylebaron** [comment]
Just realized I forgot to request the review.

File: R/class-yproject.R [hunk 1]
```diff
@@ -181,16 +182,20 @@ ys_project <- function(..., output=tempfile(fileext=".yml"),
     dups <- paste0("  ", dups)
     stop("Duplicated spec names:\n", paste0(dups,collapse = "\n"))
   }
-  output <- normalPath(output,mustWork=FALSE)
+  output <- normalPath(output, mustWork = FALSE)
   if(missing(sponsor)) {
     if(.has("sponsor", meta)) {
-      sponsor <- meta$sponsor[1]  
+      if(nchar(meta[["sponsor"]][[1]]) > 0) {
```
Comment (andersone1) [inline R/class-yproject.R:188]: @kylebaron 

The check for nchar uses double brackets around the `[[1]]` (line 188), but when it gets assigned single brackets are used (line 189). Is this intended?

Same is true for lines 195 and 196

Comment (kylebaron) [reply R/class-yproject.R:188]: Thanks, @andersone1 ; I pushed a change to we consistently use double; double and single work the same on vectors but makes sense to get it conssitent.

**Review by andersone1 — APPROVED** [review]

**Pull request merged**